packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -405,7 +406,7 @@ export class LoggingError extends Error implements ILoggingError, Omit<IFluidErr
     /**
      * Add additional properties to be logged
      */
-    public addTelemetryProperties(props: ITelemetryProperties) {
+    public addTelemetryProperties(props: ITelemetryPropertiesExt) {
```

Comment:
Let's not do this in this PR. It will not be too much work, but there are a few additional considerations so better to scope it out.  I opened https://dev.azure.com/fluidframework/internal/_workitems/edit/2802 for this.